views/keyword/search_result.html

```diff
@@ -0,0 +1 @@
+<iframe srcdoc="{{ .SearchResult.Html }}"></iframe>
```

Comment:
@junan The benefit of using `iframe` is the content inside of iframe won't ruin our layout, right?
Because what I did, the content ruined my base layout 😅

Reply:
Its does not with the `srcdoc` but it happended to me also while using `src` attribute 😄